Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,227 @@
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+It is OPTIONAL to give more information to users like this:
+
+```text
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field              | Value                                                                                                                                                        |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| access_project_ids | None                                                                                                                                                         |
+| description        | [encrypted:cipher=aes,key_size=256][replicated: 3 times] Content will be replicated three times and LUKS encryption is used.                                 |
+| id                 | d63307fb-167a-4aa0-9066-66595ea9fb21                                                                                                                         |
+| is_public          | True                                                                                                                                                         |
+| name               | hdd-three-replicas-LUKS                                                                                                                                      |
+| properties         |                                                                                                                                                              |
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+## OPTIONAL volume types
+
+In this section volume types are described, that we do not want to explicitly recommend, but find them worth listing as optional for deployers.
+
+### Availability Zones
+
+Availability Zones are not necessarily used by every deployer of a cloud. The SCS does not want do make it necessary to have AZs for a compatible deployment. But we want to encourage deployer who use multiple AZs to read the following:
```

Comment:
I read "deployer" several time throughout this standard. I doubt if this role fits best. Other standards distinguish between provider and consumer, only.  If this standard uses "deployer", this role has to be introduced beforehand.

Reply:
maybe that is from my perspective as openstack-developer: I only see the interacting roles, and not the business roles. I can edit the standard, but would also like to agree on certain terms beforehand.